### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
+
+Follow the steps below to allow users to filter a grid column by a date range:
+
+1. Add a Grid component to a page and bind the component to data.
+
+2. Set the component's [ShowFilterRow](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGrid.ShowFilterRow) property to `true` to display the filter row.
+
+3. Specify the [FilterRowCellTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridDataColumn.FilterRowCellTemplate) property of a column that displays dates. In the template, create two [Date Edit](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1) components where users can set the start and end dates of a range.
+
+4. Implement [two-way data binding](https://docs.devexpress.com/Blazor/402330/common-concepts/two-way-data-binding) between [Date](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1.Date) properties of the Date Edit components and the data fields that store endpoints of the range. Handle [DateChanged](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1.DateChanged) events of the components.
```
Comment (dmitry-eliseev-devexpress) [inline README.md]: > Handle [DateChanged](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1.DateChanged) events of the components.

In terms of logic, this sentence is a part of step 5, not 4.

```suggestion
4. Implement [two-way data binding](https://docs.devexpress.com/Blazor/402330/common-concepts/two-way-data-binding) between [Date](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1.Date) properties of the Date Edit components and the data fields that store endpoints of the range.
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
+
+5. In the `DateChanged` event handler of a Date Edit, assign the Date Edit's new value to the corresponding endpoint of the date range. If after that the start date of the range becomes larger than the end date, update the other endpoint value. Once both endpoints of the date range are set, create the filter criteria that returns whether the current value is in the range. Apply the filter criteria to the grid column.
```
Comment (dmitry-eliseev-devexpress) [inline README.md]: First -- I moved the last sentence of step 4 to step 5.

Second -- I got rid of implied synonyms ("Date Edit" and "component").

> If **after that** the start date of the range **becomes larger than** the end date, update the other endpoint value.

Yes, I understand that you meant that "after the operation I mentioned right now", however, we can say the same thing way clearer -- as a result of this operation or -- if we simplify this phrase -- just "as a result", which implies that the described behavior takes place afterwards.

> becomes larger than

Too many words to express such a simple thing. While it's correct -- our minimalistic writing style implies that we should accomplish our goals with as little words as possible if we have such a choice. And we have it here.

Therefore, "becomes larger than" == "exceeds"

Finally, the last sentence in step 5:

> create **the filter criteria that returns** whether the current value is in the range.

First. Criteria is Greek word in plural form where "criterion" is singular. While English indeed allows one to use "criterion" and "criteria" interchangeably, I believe we should properly respect singular and plural forms regardless of word origin.

Second. "Returns whether". I understand where this incomplete form comes from -- this is an often occurred Boolean property description pattern I tried fighting to no avail. Actually, the current wording indirectly implies that criteria return a Boolean value that determines whether the current value is in the range. We can exactly that but simplify the wording down to "determine". The criteria we specified just determine whether the current value is in the range.

Therefore, we should only replace "returns" with "determines" and be done with it. This change accomplishes our goals perfectly.

```suggestion
5.  Handle [DateChanged](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDateEdit-1.DateChanged) events of the Date Edit components. In a component's event handler, assign the component's new value to the corresponding endpoint of the date range. If the start date of the range exceeds the end date as a result, update the other endpoint value. Once both endpoints of the date range are set, create the filter criteria that determine whether the current value is in the range. Apply the filter criteria to the grid column.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
**Structure and Semantics**

This is a simple step-by-step instruction without significant issues. While this topic serves its purpose very well, I can't help but optimize a couple of steps. First -- the last sentence from step 4 rather belongs to step 5. This change also triggered a number of changes in the last step itself. Other than that -- the topic is almost perfect. Keep it up!

**Images**

The GIF animation is clear and perfectly illustrates the topic's main point. Well done!

**Language**

You perfected your usage of articles and the art of connecting sentences into a single cohesive whole -- and I highly commend you for that. Keep it up. But I found a number of minor issues related to wording optimization and getting rid of wordiness. You still can sharpen the precision of your wordings.

It also especially pleases me to see that you take all your previous experience into account and avoid past mistakes at all costs. Well done and I expect to see your further progress.

**Bottom Line**

A topic worthy of DevExpress documentation writing standards. I approve.
